project/Osgi.scala

```diff
@@ -50,7 +50,8 @@ object Osgi {
       "Bundle-SymbolicName" -> (bundleSymbolicName.value + ".source"),
       "Bundle-Version" -> versionProperties.value.osgiVersion,
       "Eclipse-SourceBundle" -> (bundleSymbolicName.value + ";version=\"" + versionProperties.value.osgiVersion + "\";roots:=\".\"")
-    )
+    ),
+    Keys.`package` := bundle.value
```

Comment:
Yeah. I'm happy to go either way. I picked `package` because its description says "This is typically an alias for the task that actually does the packaging.". So it felt more natural to do it this way.